components/Item/ItemGrid.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <v-row v-if="loading">
       <v-col cols="12" class="card-grid-container">
-        <skeleton-card v-for="n in 24" :key="n" />
+        <skeleton-card v-for="n in 24" :key="n" text />
```

Comment:
There's another skeleton-card for me in master in that file at L29

Reply:
Good Catch